azure/durable_functions/orchestrator.py

```diff
@@ -34,135 +27,24 @@ def __init__(self,
         :param activity_func: Generator function to orchestrate.
         """
         self.fn: Callable[[DurableOrchestrationContext], Generator[Any, Any, Any]] = activity_func
+        self.task_orchestration_executor = TaskOrchestrationExecutor()
 
-    def handle(self, context: DurableOrchestrationContext):
-        """Handle the orchestration of the user defined generator function.
+    def handle(self, context: DurableOrchestrationContext) -> str:
+        """Execute the user's orchestration.
```

Comment:
I think I slightly prefer the older language of the comment, though maybe "Execute the user defined orchestration function" is a happy medium?

Reply:
I can revert back to the old comment :)